app/src/i18n/ca/index.json

```diff
@@ -79,6 +79,7 @@
   "deleted": "Eliminada",
   "payer": "Pagador/a",
   "payee": "Cobrador/a",
+  "feedback": "Feedback",
```

Comment:
As with the spanish translation, I'd set there "Opina" (same word both in ES and CA)